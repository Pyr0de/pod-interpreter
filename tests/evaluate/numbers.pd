10+10;
10-10;
100*12;
100/23;
0.1 + 0.2;
22/7;
100 > 100;
199 > 100;
100 >= 100;
100 == 100;
100 != 100;
100 < 99;
100 < 101;
100 <= 100;

100 || 100;
100 && 100;
!100;
