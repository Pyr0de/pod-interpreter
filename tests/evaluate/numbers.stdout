ExitCode: 66
20
0
1200
4.3478260869565215
0.30000000000000004
3.142857142857143
false
true
true
true
false
false
true
true
None
None
None
